sound/Scale.java

```diff
@@ -62,21 +83,21 @@ public Integer getPitch(int whichPitch) {
     public int numPitches() {
         return scalePitches.size();
     }
-    
+
     /**
      * Create a 7-octave Pentatonic scale.
      * @return a pentationic scale
      */
     public static Scale pentatonic1() {
         Scale s = new Scale();
-        for (int i = -3; i < 4; i++) {
-            s.addPitch( Note.C      + (12 * i));
-            s.addPitch((Note.C + 2) + (12 * i));
-            s.addPitch((Note.C + 5) + (12 * i));
-            s.addPitch((Note.C + 7) + (12 * i));
-            s.addPitch((Note.C + 9) + (12 * i));
+        for (int i = START_PITCH_INTERVAL; i < END_PITCH_INTERVAL; i++) {
```

Comment:
Since both symbolic constants were set to -3 (at lines 17 and 19) this loop would never be executed.